src/main/java/com/internet/shop/controller/user/UpdateUserNameController.java

```diff
@@ -9,11 +9,11 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public class UpdateUserDetailsController extends HttpServlet {
+public class UpdateUserNameController extends HttpServlet {
     private static final Injector injector = Injector.getInstance("com.internet.shop");
     private final UserService userService =
             (UserService) injector.getInstance(UserService.class);
-    private Long id = 0L;
+    private Long id;
```

Comment:
in general we should not store state in our controllers

imagine several admins are communicating with this controller at the same time (and we have only one controller object)
what value will be here?

the best way will be to pass it from `get()` to `post()` method through JSP
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden
I can remember that you used it already